### DISCUSSION TITLE
［求助］我的安比影画满后怎么变成这样了

### DISCUSSION THREAD

**Discussion by ssj20051003** [share121/inter-knot#1602] [其他]
![1722344167240](https://github.com/user-attachments/assets/f65c7a73-8993-414c-8f0e-e315e26efd4e)
😰到底怎么办，有没有绳匠解决一下🥵

**Comment by uyfh352** [comment]
你不對勁

**Comment by Taicxxx** [comment]
**十分甚至有九分不对劲**

**Comment by ZhuYuan-148** [comment]
你不对劲
![Screenshot_20240812_140729](https://github.com/user-attachments/assets/c139fe47-b08f-463a-a65d-352fabda8261)

**Reply by ZhuYuan-148** [reply]
全弹发射！

**Comment by aizai1173** [comment]
安比是不是阳了？想帮安比擦汗 ...

**Comment by jogaaaa** [comment]
我来帮你解决
